vector/src/main/java/im/vector/app/core/platform/VectorBaseActivity.kt

```diff
@@ -249,7 +249,7 @@ abstract class VectorBaseActivity<VB : ViewBinding> : AppCompatActivity(), Maver
 
         initUiAndData()
 
-        if (vectorFeatures.isNewAppLayoutEnabled()) {
+        if (vectorPreferences.isNewAppLayoutEnabled()) {
```

Comment:
Since both `vectorFeatures.isNewAppLayoutEnabled()` and `vectorPreferences.isNewAppLayoutEnabled()` exist now, there is a risk of using the former instead of the latter, especially now that we have many open PR on the subject. Maybe rename the later to something else to force a compilation error on existing PR.

Something like `vectorFeatures.isNewAppLayoutFeatureEnabled()`. Also maybe document it to tell the developer that they may want to use `vectorPreferences.isNewAppLayoutEnabled()` instead, to take care of the labs settings.

I admit this is quite ugly...